content/academy/caching_responses_in_puppeteer.md

```diff
@@ -0,0 +1,169 @@
+---
+title: How to optimize Puppeteer by caching responses
+description: Learn why it's important to cache responses in memory when intercepting requests in Puppeteer, and how to do it.
+menuWeight: 19
+category: tutorials
+paths:
+    - caching-responses-in-puppeteer
+---
+
+# [](#caching-responses-in-puppeteer) Caching responses in Puppeteer
+
+> In the latest version of Puppeteer, the request-interception function inconveniently disables the native cache and significantly slows down the crawler. Therefore, it's not recommended to follow the examples shown in this article. Puppeteer now uses a native cache that should work well enough for most use cases.
+
+When running crawlers that go through a single website, each open page has to load all resources again (sadly, headless browsers don't use cache). The problem is that each resource needs to be downloaded through the network, which can be slow and/or unstable (especially when proxies are used).
```

Comment:
Didn't we just disprove this above? `(sadly, headless browsers don't use cache)` So I would remove it